account_analytic_parent/models/account_analytic_account.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Luxim d.o.o.
+# Copyright 2017 Matmoz d.o.o.
+# Copyright 2017 Deneroteam.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class AccountAnalyticAccount(models.Model):
+    _inherit = 'account.analytic.account'
+
+    parent_id = fields.Many2one(
```

Comment:
@pedrobaeza maybe I dont understand well your proposal but it does not fit well usabiilty:
Inactive means "Do not use" and brings confusion in your design.
Besides how do you display the View accounts if they are inactive and how you differentiate them from the actual inactive ones?

Having the type in the base modules looks cleaner and standard feature (same as in v8): later you indeed have to add domain control in the depending modules but I don't understand why you would need glue modules.